backend/sites/src/app/entities/siteProfileLandUses.entity.ts

```diff
@@ -38,13 +38,13 @@ export class SiteProfileLandUses {
   @JoinColumn([{ name: 'lut_code', referencedColumnName: 'code' }])
   lutCode2: LandUseCd;
 
-  @ManyToOne(
-    () => SiteProfiles,
-    (siteProfiles) => siteProfiles.siteProfileLandUses,
-  )
-  @JoinColumn([
-    { name: 'site_id', referencedColumnName: 'siteId' },
-    { name: 'sprof_date_completed', referencedColumnName: 'dateCompleted' },
-  ])
-  siteProfiles: SiteProfiles;
+  // @ManyToOne(
+  //   () => SiteProfiles,
+  //   (siteProfiles) => siteProfiles.siteProfileLandUses,
+  // )
```

Comment:
Remove the commented code

Reply:
need this for future in case if there will be conflict while data sync